js/_codemirror.ts

```diff
@@ -128,7 +128,7 @@ export const extensions : { [key: string]: any } = {
     'brainfuck':     brainfuck(),
     'c':             StreamLanguage.define(c),
     'c-sharp':       StreamLanguage.define(csharp),
-    'civet':         javascript,
+    'civet':         coffeeScript,
```

Comment:
I think that needs to be `StreamLanguage.define(coffeeScript)` as per lower, did it work without that?